keras_cv/layers/serialization_test.py

```diff
@@ -17,16 +17,14 @@
 from absl.testing import parameterized
 
 from keras_cv import core
-from keras_cv.layers import object_detection
-from keras_cv.layers import preprocessing
-from keras_cv.layers import regularization
+from keras_cv import layers as cv_layers
```

Comment:
Just curious - is this our convention or are you trying to avoid a name conflict here?

Since keras.layers isn't imported here I think we should be able to just call this `layers`?

Reply:
I'd like to do cv_layers.

Too confusing to call it `layers` - remember, users copy paste Keras code all over the internet.  We should encourage `cv_layers`